wls-gui-wahllokalsystem/src/types/briefwahl/BeanstandeteWahlbriefe.ts

```diff
@@ -0,0 +1,7 @@
+import type { ZurueckweisungsgrundEnum } from "@/types/briefwahl/ZurueckweisungsgrundEnum.ts";
+
+export interface BeanstandeteWahlbriefe {
+  wahlbezirkID: string;
+  waehlerverzeichnisNummer: number;
+  beanstandeteWahlbriefe: Map<string, ZurueckweisungsgrundEnum[]>;
+}
```

Comment:
Hintergedanke ist dass man leichter versteht was der Key fachlich sein soll. Da dadurch aber nicht direkt klar wird dass es sein String ist könnte man noch was am Namen machen, `WahlIDString`?

```suggestion
type WahlID = string;

export interface BeanstandeteWahlbriefe {
  wahlbezirkID: string;
  waehlerverzeichnisNummer: number;
  beanstandeteWahlbriefe: Map<WahlID, ZurueckweisungsgrundEnum[]>;
}
```

Reply:
done [593bca9](https://github.com/it-at-m/Wahllokalsystem/pull/1563/commits/593bca9909cfdacca7a611c47b002f3d36bb9106)